test/regressions/tests/Table/PaddingTable.js

```diff
@@ -45,9 +45,6 @@ function PaddingTable() {
         <Grid item xs={6}>
           <MyTable padding="default" />
         </Grid>
-        <Grid item xs={6}>
-          <MyTable padding="dense" />
```

Comment:
Should we keep this regression test? :thinking:

Reply:
Oh yes, I think that we should.

Reply:
I have added a new visual regression test.